### ISSUE TITLE
Astro C40 not working in steam, but works elsewhere in linux

### ISSUE THREAD

**Issue by zany130** [ValveSoftware/steam-for-linux#6935]
#### Your system information

* Steam client version (January 25 2020): 
* Distribution (e.g. Ubuntu): happens in Ubuntu 18.04 popos 19.10 and manjaro
* Opted into Steam client beta?: [Yes/No] happens in both (right now I am using stable)
* Have you checked for system updates?: [Yes]

#### Please describe your issue in as much detail as possible:
The Astro C40 works in retroarch and in any Linux application except steam. Using SC-controller or xboxdrv fixes the issue somewhat but introduces other issues (steam crashes random input) 

#### Steps for reproducing this issue:

1.  connect an Astro c40 controller to the computer
2. open any tool to check the controller mapping and stuff like jstest
3.  notice all buttons work as expected
4. open steam in big picture
5. notice it does not detect the controller
6. run xboxdrv(with the config I proved) or run SC-controller and add the controller to it(can be a little annoying to get setup)
[AstroC40.txt](https://github.com/ValveSoftware/steam-for-linux/files/4270590/AstroC40.txt)

7.  notice that steam now detects the controller

**Comment by zany130** [comment]
if any more information is needed please let me know

**Comment by kisak-valve** [comment]
Hello @zany130, looks like someone else encountered this issue and proposed https://github.com/ValveSoftware/steam-devices/pull/5 to fix it. Please add this udev rule to your system and retest.

**Comment by zany130** [comment]
yup that fixed it everything workes rumble and touchpad(which doesn't work on windows)

**Comment by zany130** [comment]
the controller does not work when using the steam browser or when using the big picture chat it works everywhere else. this does not happen with my Dualshock which uses the same profile

**Comment by zany130** [comment]
has this still not been merged? on a new arch install i had to add the udev rule to get the controller to work

**Comment by doidexx** [comment]
@zany130  i'm new to this and i'm having the exact same problem, can you please guide me through fixing this?!

**Comment by zany130** [comment]
> @zany130  i'm new to this and i'm having the exact same problem, can you please guide me through fixing this?!

Don't have my arch machine or my astro c40 but it should be working on a fully up to date system, if not you have to add that udev rule in the link to 
`/etc/udev/` or something like that there should be a bunch of files that start with a number and device name. You have to copy the file from the link into there with a `sudo mv filename destination`and reboot or reload udev rules check here for more info https://wiki.archlinux.org/index.php/udev

**Comment by doidexx** [comment]
Thanks, I appreciate you taking your time to help me.
